docs/DeveloperGuide.md

```diff
@@ -391,6 +391,18 @@ class PersonBuilder {
 
 All our existing tests using `PersonBuilder` will still work, and all we had to do was update definitions in `PersonBuilder`.
 
+### Show attendance
+
+Shows if a student is present on a specific day.
+
+![ShowAttendanceSequenceDiagram](images/ShowAttendanceSequenceDiagram.png)
+
+1. Parse the `showatt` command
```

Comment:
Do u wanna consider being a little more detailed about the methods invoked during `show-attend` command?